### PR TITLE
Update to PHP 8.1 and add support for Ubuntu 22.04

### PR DESCRIPTION
**Pull request by glesys-andreas**
Requirements: Ubuntu 22.04

### CODE REVIEW

File: wordpress/cloud-config.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ runcmd:
     - mysql -e "CREATE USER wordpress@localhost IDENTIFIED BY '$PASSWORD';"
     - mysql -e "GRANT ALL PRIVILEGES ON wordpress.* TO 'wordpress'@'localhost';"
     - mysql -e "FLUSH PRIVILEGES;"
+    - chmod +x /home/{{users.0.username}}
```
Comment (glesys-andreas) [inline wordpress/cloud-config.yaml:81]: As of Ubuntu 22.04 the user dir has 750 permissions by default rather than 755.

**Review by hejhansson — COMMENTED** [review]
maybe update the commit message to `Update WordPress to PHP 8.1 and add support for Ubuntu 22.04` or split it into two commits: `Update WordPress to PHP 8.1` and `Add Ubuntu 22.04 support for WordPress`.

**Comment by glesys-andreas** [comment]
@hejhansson Updated the commit with the new message

**Review by hejhansson — APPROVED** [review]

**Review by olvert — APPROVED** [review]

**Review by hejhansson — APPROVED** [review]

File: wordpress/cloud-config.yaml (hunk 1 above)
Comment (brother) [reply wordpress/cloud-config.yaml:81]: That poses an interesting problem indeed.
What's harder to grasp - all newly created users must be adapted manually or change the default umask for the system (deviating from upstream default). I am not sure what path to choose really. From a support staff perspective it's probably better to not change the default and have "check user dir permissions" as the first bullet point in troubleshooting.

**Review by emil-nasso — APPROVED** [review]

**Pull request merged**